eventsource/resources/status-error.py

```diff
@@ -1,4 +1,12 @@
 def main(request, response):
   status = (request.GET.first("status", "404"), "HAHAHAHA")
   headers = [("Content-Type", "text/event-stream")]
-  return status, headers, "data: data\n\n"
+
+  # According to RFC7231, HTTP responses bearing status code 204 or 205 must
+  # not specify a body.
```

Comment:
Could you please update this with a reference to the issue instead? What the RFC says is conforming does not matter much with respect to how implementations have to handle the situation where it happens anyway.